model/src/channel/permission_overwrite.rs

```diff
@@ -127,6 +154,39 @@ mod tests {
         assert_eq!(serde_json::to_string_pretty(&overwrite).unwrap(), input);
     }
 
+    #[test]
+    fn test_blank_overwrite() {
+        // Test integer deser used in guild templates.
+        let blank_overwrite = PermissionOverwrite {
+            allow: Permissions::CREATE_INVITE,
+            deny: Permissions::KICK_MEMBERS,
+            kind: PermissionOverwriteType::Member(UserId(0)),
+        };
+
+        let input = r#"{
+  "allow": "1",
+  "deny": "2",
+  "id": 0,
+  "type": 1
+}"#;
+
+        let output = r#"{
```

Comment:
Can these use serde-test? We want to test for all serialization formats and not just JSON